tests/translations/tests/fixtures/t.filter.html.twig

```diff
@@ -1,6 +1,6 @@
             {% if period.probabilityOfPrecipitation and period.probabilityOfPrecipitation > 1 %}
             <p class="text-gray-50 font-body-xs margin-top-05 margin-bottom-0">
-            {{period.probabilityOfPrecipitation}}% {{ "daily-forecast.text.chance-precip.01" | t }}
+            {{ "daily-forecast.text.chance-precip.02" | t({ "@percent": period.probabilityOfPrecipitation }) }}
```

Comment:
First time using the versioning on these strings. It will be better to include the number in the string, since some languages prefix the number with letters or symbols or don't use "%".